scope3_methodology/corporate/yaml_keys.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+""" ATP Model Yaml Keys """
+from dataclasses import dataclass
+
+
+@dataclass
+class CorporateKeys:
+    number_of_employees: str = "employees"
+    corporate_emissions_mt_per_month: str = "corporate_emissions_mt_per_month"
+    office_emissions_mt_per_employee_per_month: str = "office_emissions_mt_per_employee_per_month"
+    travel_emissions_mt_per_employee_per_month: str = "travel_emissions_mt_per_employee_per_month"
+    overhead_emissions_mt_per_employee_per_month: str = (
+        "overhead_emissions_mt_per_employee_per_month"
+    )
+    commuting_emissions_mt_per_employee_per_month: str = (
+        "commuting_emissions_mt_per_employee_per_month"
+    )
+    datacenter_emissions_mt_per_employee_per_month: str = (
+        "datacenter_emissions_mt_per_employee_per_month"
+    )
+
+    def get_default_keys(self) -> set[str]:
+        return {
+            self.corporate_emissions_mt_per_month,
```

Comment:
this key isn't being used in the __init__ method of `corporate_defaults`. is that intentional?